docs/t-digest-paper/histo.tex

```diff
@@ -56,10 +56,10 @@ \section{The Algorithm}
 \KwOut{final set $C=[c_1 \ldots c_m]$ of weighted centroids} 
 \For{$(x_n, w_n) \in X$} {
   $z = \min | c_i.\mathtt{mean} - x |$\;
-  $S = \lbrace c_i  :  |c_i.\mathtt{mean} - x| = z \wedge c_i.\mathtt{count} + 1 \le 4n\delta q(c_i) (1-q(c_i)) \rbrace $\;
+  $S = \lbrace c_i  :  |c_i.\mathtt{mean} - x| = z \wedge c_i.\mathtt{count} + w_n \le 4n\delta q(c_i) (1-q(c_i)) \rbrace $\;
```

Comment:
We need to exclude Centroids that have less than the necessary space left. `1` is fine when the data has weight `1`, but in general (like the algorithm here states), we assume a weight of `w_n`.